scss/components/_case-activity-count.scss

```diff
@@ -2,13 +2,12 @@
   border: 1px solid $gray-light;
   border-radius: 2px;
   box-shadow: $box-shadow-tooltip;
-  margin-left: 10px;
   padding: 0;
   white-space: nowrap;
-  z-index: 5;
+  z-index: 10;
```

Comment:
What is the need of changing the z-index?

Reply:
This way the popup comes above of the case list pager.

Reply:
I agree, but few points.
Why z-index: 5 was not working 
and If it tends to fix something. Should we add it in the PR description ?

Reply:
Added to the PR description